docs/_docs/structure.md

```diff
@@ -144,6 +144,7 @@ An overview of what each of these does:
           These are sass partials that can be imported into your <code>main.scss</code>
           which will then be processed into a single stylesheet
           <code>main.css</code> that defines the styles to be used by your site.
+          Learn <a href="/docs/assets/">how to work with assets</a>. 
```

Comment:
Attaching `relative_url` to anchor references will ensure that the link doesn't break and return 404 error if there's a change in the site's baseurl in the future.
```suggestion
          Learn <a href="{{ '/docs/assets/' | relative_url }}">how to work with assets</a>. 
```